packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -25,23 +31,16 @@ import { SquadsList } from './SquadsList';
 import { Button } from '../buttons/Button';
 import PlusIcon from '../icons/Plus';
 import { ProfilePicture } from '../ProfilePicture';
+import { useLazyModal } from '../../hooks/useLazyModal';
+import { LazyModal } from '../modals/common/types';
+import { Squad } from '../../graphql/squads';
 
 const UserSettingsModal = dynamic(
   () =>
     import(
       /* webpackChunkName: "userSettingsModal" */ '../modals/UserSettingsModal'
     ),
 );
-const SquadsBetaModal = dynamic(
```

Comment:
@el I changed this, however I think this might introduce the modal lag I was talking about before.
We can see how it behaves on preview.